static/sass/_pattern_p-card.scss

```diff
@@ -1,29 +1,4 @@
 @mixin p-charmhub-card {
-  .p-card--bundle {
```

Comment:
removing this css I think messes up the styling for the bundle icon on the bundle details page.

Reply:
Hmm, I'll check that when get to the details page - not sure the details page should be borrowing styles from the card pattern!

Reply:
You are 100% right - it's for the full bundle cards, not the homepage cards. Readded